### PR TITLE
Update of output and plugin documentation

### PR DESCRIPTION
**Pull request by jurvo**

### CODE REVIEW

**Comment by lukasmerten** [comment]
Please check the plugin part, too.

**Comment by jurvo** [comment]
checked the plugin and fixed a small error in the python script

File: doc/pages/example_notebooks/extending-CRPropa/extending-CRPropa.ipynb [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    "class MySourceFeature(SourceFeature):\n",
+    "\t\"\"\" Set the initial energy to 10 EeV \"\"\"\n",
+    "\tdef __init__(self):\n",
+    "\t\tSourceFeature.__init__(self)\n",
+    "\n",
+    "\tdef prepareParticle(self, particleState):\n",
+    "\t\tparticleState.setEnergy(10 * EeV)\n",
+    "\n",
+    "# The source feature has to be created outside of the class attribute\n",
+    "# s.add(MySourceFeature()) wil NOT work! (SWIG issue)\n",
```
Comment (lukasmerten) [inline doc/pages/example_notebooks/extending-CRPropa/extending-CRPropa.ipynb]: Here is a typo "will"

File: include/crpropa/module/HDF5Output.h [hunk 2]
```diff
@@ -22,7 +22,7 @@ const size_t propertyBufferSize = 1024;
 /**
  @class HDF5Output
  @brief Output to HDF5 Format.
-
+In the base class is an overview of the possible columns.
```
Comment (lukasmerten) [inline include/crpropa/module/HDF5Output.h]: Maybe better: "The baseclass gives an overview of possible columns."

**Comment by lukasmerten** [comment]
Looks good apart from the small comments.

**Review by lukasmerten — APPROVED** [review]

**Pull request merged**